docs/products/postgresql/howto/visualize-grafana.rst

```diff
@@ -0,0 +1,41 @@
+Visualise PostgreSQL Data with Grafana
+======================================
+
+PostgreSQL can hold a wide variety of types of data, and creating visualisations helps gather insights on top of raw figures. Luckily, Aiven can set up the Grafana and the integration between the two services for you.
+
+
+Integrate PostgreSQL and Grafana
+--------------------------------
+
+1. On the service overview page for your PostgreSQL service, go to **Manage Integrations** and choose the **datasource** option.
+
+2. Choose either a new or existing Grafana service.
+
+   - A new service will ask you to select the cloud, region and plan to use. You should also give your service a name. The service overview page shows the nodes rebuilding, and then indicates when they are ready.
+   - If you're already using Grafana on Aiven, you can integrate your PostgreSQL as a data source for that existing Grafana.
+
+3. On the service overview page for your Grafana service, click the "Service URI" link. The username and password for your Grafana service is also available on the service overview page.
+
+Now your Grafana service is connected to PostgreSQL as a data source and you can go ahead and visualise your data.
+
+Visualize PostgreSQL data in Grafana
+------------------------------------
+
+In Grafana, create a new dashboard and add a panel to it.
+
+The datasource dropdown shows ``--Grafana--`` by default, but your PostgreSQL service will be listed here with a Prometheus logo. Prometheus is managing the communication between PostgreSQL and Grafana.
```

Comment:
Does PostgreSQL really show as the prometheus content? Or was this article copy/pasted from the M3DB one? Can you check? Also let's not copy/paste within this site, it's fine from the old site as a source but we should either re-use content, or write fresh content for the various sections here.

Reply:
Yep my mistake, I clearly didn't check this page enough